README.md

```diff
@@ -119,6 +119,23 @@ pip install -e .
 
 ### Running dot
 
+#### Graphical interface
+
+To use the graphical interface of dot run
+
+```bash
+conda activate dot
+python ./src/dot/ui/ui.py
+```
+
+Watch the following demo video for better understanding of the interface
+
+<p align="center">
+<img src="./assets/gui_dot_demo.gif" width="500" height="406"/>
+</p>

```

Comment:
the demo video is not sufficient enough for usage instructions. please add some instructions here on how to use the available fields.